examples/optimize_hyperparams_optuna.py

```diff
@@ -30,7 +31,7 @@ def moving_average(array, n_mov=4):
 if not os.path.exists('downloads/prompt.root'):
     os.system('curl -L https://cernbox.cern.ch/index.php/s/98tSXndX0VhUMbi/download --output downloads/prompt.root')
 
-MOV_AVER_N = 4
+N_JOBS = 40  # set number of jobs to be executed in parallel
```

Comment:
```suggestion
N_JOBS = 4  # set number of jobs to be executed in parallel
```
I would put a number sensible for a laptop